### DISCUSSION TITLE
Question/feature request - Multiple ethernets

### DISCUSSION THREAD

**Discussion by TimyIsCool** [RaspAP/raspap-webgui#1399] [Q&A]
Hi, I have two ethernet port (onboard and USB to ethernet) and I want one to use the pi network (same as the wireless network) but also want internet access via the USB ethernet but still go through the pi
would that be possible?

**Comment by billz** [comment]
This is possible with a software bridge, yes. RaspAP will create a bridged AP between eth0 and wlan0, but it does not support bridging  usb0 (or eth1). This seems like an edge case so we're more likely to focus our efforts elsewhere. You're welcome to improve this project by opening a PR, however.

**Reply by he-so** [reply]
> RaspAP will create a bridged AP between eth0 and wlan0

Reading this, I am wondering: Can I use the Pi as a client in my existing WiFi network, and attach one more ethernet device to eth0 that uses the existing DHCP in the Wifi? Like a WiFi<->ETH Layer 2 bridge?

**Reply by billz** [reply]
Yes, this configuration is enabled with [WLAN routing](https://docs.raspap.com/wlanrouting/).
